### ISSUE TITLE
catalyst: Change color and/or logo

### ISSUE THREAD

**Issue by NunoAlexandre** [centrifuge/centrifuge-chain#788]
The goal is to be able to easily visually distinguish between Catalyst (on Rococo) and Centriuge on Polkadot.

**Comment by mustermeiszer** [comment]
Still relevant @NunoAlexandre ?

**Comment by NunoAlexandre** [comment]
@mustermeiszer yes! I have no idea how to do this tho. @mikiquantum do you know? 🤔

**Comment by mikiquantum** [comment]
We need to create a PR against polkadotjs/apps adding catalyst color schemes & logo. Let's leave it open for now.
